templates/account/signup.html

```diff
@@ -85,4 +105,41 @@ <h2 class="text-xl text-center">Or Sign Up with Email</h2>
 
   </div>
 
+  <script>
+
+    function toggleButtonsOff(state) {
```

Comment:
```
function toggleButtons(disable = true) { ... }
```
**suggestion:** How about renaming to `toggleButtons`? The name better reflects the function's purpose since it can both enable/disable buttons
**note:** We could use the `disable` param with the default being true

Reply:
discussed. current naming is more descriptive to it's function.